app/src/main/res/layout/widget_4x1.xml

```diff
@@ -54,5 +54,14 @@
 	        android:src="@drawable/content_new_holo_light"
 	        android:background="@drawable/appwidget_bg_clickable"
 	        android:contentDescription="@string/description_add_transaction_icon"/>
+
+	   <ImageButton android:id="@+id/btn_view_account"
```

Comment:
@czan What is the purpose of this button? Tapping the widget already opens the account does it not?